db/models/applications.js

```diff
@@ -2,6 +2,7 @@
 'use strict';
 const moment = require('moment');
 const { Model, Op } = require('sequelize');
+const { APPLICATION_PROGRESS_STATES } = require('../../controllers/apply/awards-for-all/constants');
```

Comment:
Importing this from specific router code feels like a bit of a smell. Can we define this in this file as an ENUM?

Reply:
I explored around the ENUMs when I was working on the Application Expiry piece but always found inconsistencies around whether its supported or not similar to many other types like array etc. The [doc here](https://sequelize.org/master/class/lib/data-types.js~ENUM.html) says it's only compatible with postgres.

Reply:
Oh, interesting, we are using ENUM on the survey model but I guess it just doesn't do what we think it does! Fair enough, keep it like this. Still reckon the constants should live in here rather than in the awards for all constants but otherwise looks good.